module/VuFind/src/VuFind/ILS/Driver/Aleph.php

```diff
@@ -942,7 +932,7 @@ public function getHolding($id, array $patron = null, array $options = [])
             }
             $matches = [];
             $dueDateWithStatusRegEx = "/([0-9]*\\/[a-zA-Z]*\\/[0-9]*);([a-zA-Z ]*)/";
-            $dueDateRegEx = "/([0-9]*\\/[a-zA-Z]*\\/[0-9]*)/";
+            $dueDateRegEx = "/([0-9]*\\/[a-zA-Z0-9]*\\/[0-9]*)/";
```

Comment:
I'm guessing that this change is needed because of some kind of date formatting difference between Aleph systems -- i.e. dates may be 06/Jun/2021 in some places, and 06/06/2021 in others. If that's the case (and maybe it's not, because I'm just guessing), does the same change need to be made to $dueDateWithStatusRegEx above, in order to correctly extract request status where appropriate? Or is all of that obsolete too?

Reply:
Yes, you are right, fixed.